gulpfile.js

```diff
@@ -15,12 +15,14 @@ import fetch from './tasks/fetch.js';
 gulp.task(
   'default',
   gulp.series(
+    bake,
     clean,
-    styles,
+    clear,
     copy,
+    fetch,
     gulp.parallel(lint, scripts, images),
     nunjucks,
-    bake
+    styles
```

Comment:
Alpha-sorted the commands and added missing functions that were imported at the top of the file.